packages/fluentui/react-northstar/src/components/Tree/TreeItem.tsx

```diff
@@ -265,6 +266,18 @@ export const TreeItem: ComponentWithAs<'div', TreeItemProps> & FluentComponentSt
     _.invoke(props, 'onClick', e, props);
   };
 
+  const handleKeyDown = (e: React.KeyboardEvent) => {
+    e.preventDefault();
+    e.stopPropagation();
+    if (e.key.match(/\S/) && e.key !== '*') {
```

Comment:
is there are requirement to support more than A-Z characters ? I thought the requirement for key search was an a11y requirement in which case A-Z is best practice

Reply:
It's true that the [aria example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/treeview-2a.html) specifies only a-z/A-Z.
But the js given by aria on the same page uses the code below to match the 1st letter. So it is not a strict match to a-z/A-Z
```js
function isPrintableCharacter (str) {
    return str.length === 1 && str.match(/\S/);
}
```
I talked to @jurokapsiar, we decide to go with supporting more characters.

Reply:
There's also plan to support IME input (for example Chinese input). Although I have 0 idea how to do that ...

Reply:
If we want to support more keyboard characters that's fine then.

I orignially thought the motivation was for other languages like chinese which I am doubtful which we will do